.editorconfig

```
@@ -6,5 +6,6 @@ indent_style = space
 indent_size = 2
 end_of_line = lf
 insert_final_newline = true
-trim_trailing_whitespace = true
+# this breaks test cases, see https://github.com/editorconfig/editorconfig-vscode/issues/359
+trim_trailing_whitespace = false
```

The best way around this is to do this:
```js
testCase: `
  code${' '}
`,
```

This is better than just adding a whitespace because it clearly shows there's a trailing space as opposed to this:
```js
testCase: `
  code     
`,
```
Did you know there was 5 trailing spaces there? bet you didn't without highlighting the code block.

```suggestion
trim_trailing_whitespace = true
```

---

Ah - that is clever. Thanks!